src/book/garden-lattice/understanding.md

```diff
@@ -0,0 +1,113 @@
+Its presence is a commitment to maintaining a respectful and inclusive environment for all participants.
+It also may provide an outline for acceptable participation when it comes to conflicts of interest or similar.
+"CoCs are a place to express and define shared community values in OSS projects, an important governance document for moderation, and an indicator of project inclusivity and safety, especially for vulnerable contributors" {cite:p}`damian_codes_2024`
+A project community benefits from a shared understanding of respectful and supportive interactions, ensuring that all members can contribute positively, much like a garden requires a harmonious ecosystem to thrive.
+
+For further reading and examples of `CODE_OF_CONDUCT` files, see the following:
+
+- [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/)
+- [Example: Python Software Foundation Code of Conduct](https://policies.python.org/python.org/code-of-conduct/)
+- [Example: Rust Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct)
+
+### LICENSE
+
+A `LICENSE` file specifies the terms under which the project's code can be used, modified, and shared.
+
+> "When you make a creative work (which includes code), the work is under exclusive copyright by default. Unless you include a license that specifies otherwise, nobody else can copy, distribute, or modify your work without being at risk of take-downs, shake-downs, or litigation. Once the work has other contributors (each a copyright holder), “nobody” starts including you." {cite:p}`choosealicense_nopermission`.
+> The presence of a `LICENSE` file is crucial for legal clarity, encourages the responsible use or distribution of the project, and is considered an indicator of project maturity{cite:p}`deekshitha_rsmm_2024`.
+
+Similar to how a garden's health depends on understanding the rules of nature and respecting boundaries, a project's sustainability relies on clear licensing that protects both the creators and users, fostering a culture of trust and collaboration.
+
+For further reading and examples of `LICENSE` files, see the following:
+
+- [OSF: Licensing](https://help.osf.io/article/148-licensing)
+- [GitHub: Licensing a repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)
+- [Choosealicense.com](https://choosealicense.com/)
+- [SPDX License List](https://spdx.org/licenses/)
+
+## Project documentation
+
+Common documentation files like `README`'s, `CONTRIBUTING` guides, and `LICENSE` files are only a start towards more specific project information.
+Comprehensive project documentation is akin to a detailed gardener's notebook for a well-maintained project, illustrating how the project may be used and guiding philosophy.
+This includes in-depth explanations of the project's architecture, practical usage examples, API references, and development workflows.
```

Comment:
prob should define API

Reply:
Thanks, will revise here based on this and the other comment.